packages/cli/src/codegen.ts

```diff
@@ -228,27 +228,31 @@ function genEventIndex(contract: Contract): string {
   return `eventIndex = { ${defs} }`
 }
 
-function genConsts(contract: Contract): string {
-  const constants = genConstants(contract.constants)
-  const enums = genEnums(contract.enums)
+function genLocalConsts(contract: Contract): string {
+  const constants = genConstants(contract.constants, true)
+  const enums = genEnums(contract.enums, true)
   const constDefs = constants.concat(enums)
   if (constDefs.length === 0) {
     return ''
   }
   return `consts = { ${constDefs.join(',')} }`
 }
 
-function genConstants(constants: Constant[]): string[] {
-  return constants.map((constant) => `${constant.name}: ${nodeValToString(constant.value)}`)
+function genConstants(constants: Constant[], isLocal: boolean): string[] {
+  const prefix = isLocal ? '' : 'export const '
+  const separator = isLocal ? ': ' : ' = '
+  return constants.map((constant) => `${prefix}${constant.name}${separator}${nodeValToString(constant.value)}`)
 }
 
-function genEnum(enumDef: Enum): string {
+function genEnum(enumDef: Enum, isLocal: boolean): string {
+  const prefix = isLocal ? '' : 'export const '
+  const separator = isLocal ? ': ' : ' = '
   const fields = enumDef.fields.map((field) => `${field.name}: ${nodeValToString(field.value)}`).join(',')
-  return `${enumDef.name}: { ${fields} }`
+  return `${prefix}${enumDef.name}${separator}{ ${fields} }`
```

Comment:
Let's use typescript enum: https://www.typescriptlang.org/docs/handbook/enums.html

Reply:
We convert `U256` and `I256` to `bigint`, but typeScript does not support `bigint` as an enum constant: https://github.com/microsoft/TypeScript/issues/37783

Reply:
Interest. Let's use object then